### ISSUE TITLE
How to delete a history entry on macOS?

### ISSUE THREAD

**Issue by devnoname120** [cantino/mcfly#293]
👉 Press <kbd>fn</kbd> <kbd>option ⌥</kbd> <kbd>F1</kbd> instead of just <kbd>F1</kbd>.

This was annoying for me to find out, so I put it here for future internet travelers.

**Comment by navazjm** [comment]
It also depends on how you have macOS setup. For myself, I updated my system preferences so that the top row of keys work as standard function keys without holding the fn key. Therefore, I only have to press F1 to delete a history entry. Also note that I am still on macOS Monterey 12.6. 

[article on how to change your preferences](https://support.apple.com/en-us/HT204436)

**Comment by devnoname120** [comment]
Thanks! I don't do that as I like using these keys for brightness, switching apps, etc.

I wish the hotkeys were configurable in McFly, but I will probably not be able to budget time for that I'm already overwhelmed with my side projects.

**Comment by navazjm** [comment]
> Thanks! I don't do that as I like using these keys for brightness, switching apps, etc.
> 

Changing the system preferences mention above just changes the priority of the function key row. You can still control brightness by pressing fn + f key

**Comment by devnoname120** [comment]
@navazjm You are right but as far as I'm concerned McFly is literally the only piece of software that requires me to use <kbd>F*</kbd> keys. So in my case making function keys such as brightness control harder to access is a bad tradeoff.
